nixos/modules/services/misc/etebase-server.nix

```diff
@@ -5,8 +5,8 @@ with lib;
 let
   cfg = config.services.etebase-server;
 
-  pythonEnv = pkgs.python3.withPackages (ps: with ps;
-    [ etebase-server daphne ]);
+  pythonEnv = cfg.pythonPackage.withPackages (ps: with ps;
+    [ cfg.package daphne ]);
```

Comment:
I have my doubts that this is creating a proper env. We should add this in the package instead.

Reply:
This has been here before this PR, which fixes a non-functional service. Happy to change this in a follow up, it should also use uwsgi as upstream.

It creates a proper venv insofar as it works. (see e.g. nixos tests)